common.mk

```diff
@@ -341,6 +341,17 @@ miniruby$(EXEEXT): config.status $(ALLOBJS) $(ARCHFILE)
 
 objs: $(ALLOBJS)
 
+$(COROUTINE_OBJ).json:
+	$(Q) echo > $@
+
+%.o.json: %.d
+	$(Q) echo > $@
```

Comment:
How are `.d` files related to this?
Anyway, GNU make specific rules should be in defs/gmake.mk.

Reply:
I think @jhawthorn was seeing errors with DTrace, so I made a dummy rule to create a blank file.  I've removed this because I can't reproduce it.  I also moved everything to `defs/gmake.mk`.